demo/search_ranges.R

```diff
@@ -7,6 +7,7 @@ library(RXapian)
 db<-c("path/to/db")
 
 vrp1<-list(type="proc.num", value=c(0), check.str="mm", flags=c("RP_SUFFIX"))
+#If a Xapian version earlier than 1.3.6 is used replace vrp1 with: vrp1<-list(type="proc.num", value=c(0), check.str="mm", prefix=FALSE)
```

Comment:
For pre-1.3.6, you could explicitly check `flags=c("RP_SUFFIX")` yourself, to provide the same API to R users?